api4/cloud.go

```diff
@@ -254,3 +261,68 @@ func confirmCustomerPayment(c *Context, w http.ResponseWriter, r *http.Request)
 
 	ReturnStatusOK(w)
 }
+
+func getInvoicesForSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
+	if c.App.Srv().License() == nil || !*c.App.Srv().License().Features.Cloud {
+		c.Err = model.NewAppError("Api4.getInvoicesForSubscription", "api.cloud.license_error", nil, "", http.StatusNotImplemented)
+		return
+	}
+
+	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
+
+	products, appErr := c.App.Cloud().GetInvoicesForSubscription()
```

Comment:
Should this be called `invoices`?
```suggestion
	invoices, appErr := c.App.Cloud().GetInvoicesForSubscription()
```